cape_privacy/coordinator/auth/api_token.py

```diff
@@ -0,0 +1,52 @@
+from cape_privacy.coordinator.utils import base64
+
+SECRET_BYTES = 16
+VERSION = b"\x01"
+
+
+class APIToken:
+    """Represents an API token used to authenticate with the coordinator.
+
+    The format is: <token_id>,<base64 string>
```

Comment:
```suggestion
    The format is: <token_id>, <base64 string>
```

Reply:
Hmm that's literally the format of the token "<token_id>" followed by a comma "," and then the "base64 string". Can you think of a better way to represent this?

Reply:
Yeah I was just adding a space after the comma